command/info.go

```diff
@@ -23,8 +24,9 @@ var Info = cli.Command{
 	`,
 	Action: func(c *cli.Context) error {
 		_, cfg := setup(c)
-		fmt.Println("Current target : " + cfg.URL)
-		fmt.Println("Current user : " + cfg.User)
+		fmt.Println("Target: " + cfg.URL)
+		fmt.Println("User: " + cfg.User)
+		fmt.Println("Version: " + c.App.Version)
```

Comment:
I guess this is the CLI version, could we describe it on the output something like `CLI version` instead of `version` it can be confusing so you don't know if is cli or "ernest-server" version